modules/openxr/extensions/openxr_composition_layer_extension.cpp

```diff
@@ -233,6 +242,16 @@ XrCompositionLayerBaseHeader *OpenXRViewportCompositionLayerProvider::get_compos
 		} break;
 	}
 
+	if (extension_property_values_changed) {
+		extension_property_values_changed = false;
+
+		void *next_pointer = nullptr;
+		for (OpenXRExtensionWrapper *extension : OpenXRAPI::get_registered_extension_wrappers()) {
+			next_pointer = extension->set_viewport_composition_layer_and_get_next_pointer(composition_layer, extension_property_values, next_pointer);
```

Comment:
This differs slightly from the pattern we usually use; the returned pointer could be null so you to check it first before passing it forward.

See https://github.com/godotengine/godot/blob/79de2eaae7fb3bb8f13031ac2ec9b19128b6f471/modules/openxr/openxr_api.cpp#L328 for reference

Reply:
Ah, that makes sense. Extensions are _supposed_ to return the `p_next_pointer` if they don't add any of their own stuff, but that would protect against extensions that don't do the right thing. I'll update the PR!

Reply:
Fixed in my latest push!